src/cypress/config/ubkg.js

```diff
@@ -0,0 +1,37 @@
+// https://docs.google.com/spreadsheets/d/1RMtWkvK6KSfLrBpKOXEKxYxNoOV1Mc6AB2OZPc2ZOhM/edit#gid=0
+const UBKG = {
+    endpoints: {
+        default: 'valueset?parent_sab=SENNET&parent_code={code}&child_sabs=SENNET',
```

Comment:
Similarly, remove SenNet specific calls and results check, use the ones for HuBMAP.